team-validator.js

```diff
@@ -407,28 +407,27 @@ class Validator {
 					// They're probably incompatible if all potential fathers learn more than
 					// one limitedEgg move from another egg.
 					let validFatherExists = false;
-					for (let i = 0; i < lsetData.sources.length; i++) {
-						if (lsetData.sources[i].charAt(1) === 'S' || lsetData.sources[i].charAt(1) === 'D') continue;
-						let eggGen = parseInt(lsetData.sources[i].charAt(0));
-						if (lsetData.sources[i].charAt(1) !== 'E' || eggGen === 6) {
+					for (const learned of lsetData.sources) {
```

Comment:
``source`` instead of ``learned`` maybe?

Reply:
yeah I was unsure of the naming of that section... some of the other loops already used `learned` but IDK, is `source` the right one?

Reply:
Either is fine; I don't care too much.

Reply:
eh, going to keep it at `learned` because there is a nested loop that makes more sense for `source` IMO

btw, I hope I didn't screw something up by using for-of (I had to use `for (let learned of lsetData)` a few times because of needing to change the value of the variable within the loop) in that massive nested-loop mess that is `team-validator.js` (thanks Game Freak!)

Reply:
Having read the rest of the code, I think this one should be `source`

Reply:
ok